File: examples/python/active_voice_rewriter.py
from trainloop_llm_logging import collect, trainloop_tag

collect(flush_immediately=True)

from ai_request import make_ai_request

if __name__ == "__main__":
    passive = "The quarterly report was prepared by the finance team."
    prompt = f"""Rewrite the sentence below in ACTIVE voice, preserving meaning.  
Sentence: "{passive}"
Only output the rewritten sentence."""
    headers = trainloop_tag("active-voice")
    print(make_ai_request(prompt, extra_headers=headers))

File: examples/typescript/activeVoiceRewriter.ts
// Notice that trainloop logging should be setup via --require flag in package.json script
import { trainloopTag, collect } from 'trainloop-llm-logging';
collect(true);

import { makeAiRequest } from './aiRequest';

// Example usage
async function main() {
    const passive = "The quarterly report was prepared by the finance team.";
    const promptText = `Rewrite the sentence below in ACTIVE voice, preserving meaning.  
Sentence: "${passive}"
Only output the rewritten sentence.`;

    // Tag this request for the active voice evaluation suite
    const headers = trainloopTag("active-voice");
    const response = await makeAiRequest(promptText, "gpt-4o-mini", 500, headers);
    
    if (response) {
        console.log("AI Response:", response);
    }
    process.exit(0);
}

if (require.main === module) {
    main().catch(console.error);
} 